data-builder/build.py

```diff
@@ -33,6 +34,8 @@ def build_data(clearCache, discordURL):
     parse_minor_artifacts()
 
     parse_item_types()
+
+    parse_quests()
```

Comment:
This was a small change to call the `parse_quests` method you had already created so that the parser will continue to update the raid quest list.